pennylane/workflow/execution.py

```diff
@@ -61,23 +61,42 @@
     "tensorflow",
 }
 
-INTERFACE_MAP = {
-    None: "Numpy",
-    "auto": "auto",
-    "autograd": "autograd",
-    "numpy": "autograd",
-    "scipy": "numpy",
-    "jax": "jax",
-    "jax-jit": "jax",
-    "jax-python": "jax",
-    "JAX": "jax",
-    "torch": "torch",
-    "pytorch": "torch",
-    "tf": "tf",
-    "tensorflow": "tf",
-    "tensorflow-autograph": "tf",
-    "tf-autograph": "tf",
-}
+SUPPORTED_INTERFACE_USER_INPUT = Literal[

```

Comment:
For type hints, for the sake of consistency, can you change this to use camel-case instead? Same for `SUPPORTED_DIFF_METHODS` in `qnode.py`.
```suggestion
SupportedInterfaceUserInput = Literal[

```